luasec/include/options.h

```diff
@@ -2,148 +2,21 @@
 #define LSEC_OPTIONS_H
 
 /*--------------------------------------------------------------------------
- * LuaSec 0.6
- * Copyright (C) 2006-2016 Bruno Silvestre
+ * LuaSec 1.3.2
+ *
+ * Copyright (C) 2006-2023 Bruno Silvestre
  *
  *--------------------------------------------------------------------------*/
 
-#include <openssl/ssl.h>
-
-/* If you need to generate these options again, see options.lua */
-
-/* 
-  OpenSSL version: OpenSSL 1.1.0e
-*/
+#include "compat.h"
 
-struct ssl_option_s {
+struct lsec_ssl_option_s {
   const char *name;
-  unsigned long code;
+  uint64_t code;
```

Comment:
Explicitly set `uint64_t` type because for 32bit Android unsigned long == 32 bit, but stored 64bit integer which leads to compilation error.